lib/run.go

```diff
@@ -131,9 +131,16 @@ func (a *ACBuild) Run(cmd []string, insecure bool) (err error) {
 	}
 
 	if man.App != nil {
+		setPath := false
 		for _, evar := range man.App.Environment {
+			if evar.Name == "PATH" {
```

Comment:
Setting the `PATH` for acbuild will probably be more useful than setting the `PATH` for rkt, since the AppC spec forces you to provide an absolute path to the binary you're exec'ing, and `acbuild run` doesn't. I still think it'd be useful to override the default in rkt though, since I could probably come up with some edge case related to `nix2aci` where having a weird `PATH` would be useful.

I think rkt should do what I'm doing here. I'll file an issue against rkt to see what people think.